core/src/main/java/hudson/Util.java

```diff
@@ -568,6 +568,20 @@ public static boolean isRelativePath(String path) {
     }
 
     /**
+     * A check if a file path is a descendant of a parent path
+     * @param forParent the parent the child should be a descendant of
+     * @param potentialChild the path to check
+     * @return true if so
+     * @throws IOException potentially from {@link Files#isSameFile(Path, Path)}
```

Comment:
no, well maybe. It might not be throwing that any more :D thanks for catching it.